### PR TITLE
Update BLorient757.xml

### PR DESCRIPTION
**Pull request by DariaElagina**

### CODE REVIEW

File: LondonBritishLibrary/orient/BLorient757.xml [hunk 1]
```diff
@@ -31,23 +31,126 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                        </msItem>
+                        <msItem xml:id="ms_i2">
+                          <locus from="131r"></locus>
+                          <title type="complete" ref="LIT2050Nagara"></title>
+                          <textLang mainLang="gez"></textLang>
+
+                          <msItem xml:id="ms_i2.1">
+                            <locus from="131r"></locus>
+                            <title type="complete" ref="LIT2050Nagara#IsaiahOfScete" xml:lang="en">
+                              Extract from Isaiah of Scete
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient757.xml:54]: Maybe "excerpt" is better? Or nothing has to be written in the title, if the work record is elaborated.

File: LondonBritishLibrary/orient/BLorient757.xml [hunk 2]
```diff
@@ -56,59 +159,90 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
-                                
-                                
+
+
                             </supportDesc>
                             <layoutDesc>
-                                <layout columns="2" writtenLines="20 22" cert="high">
-                           <note>except <locus from="134" to="142"/> where the number is larger.</note>
+                                <layout columns="2" writtenLines="20 22">
+                           <note>On <locus from="134" to="142"/> the number of
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient757.xml:174]: I would maybe have a second layout with this locus range and the note.

File: LondonBritishLibrary/orient/BLorient757.xml [hunk 3]
```diff
@@ -31,23 +31,126 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
                         <summary/>
-                        
-                        
-                        
-                        
+                        <msItem xml:id="ms_i1">
+                          <locus from="4r"></locus>
+                          <title type="complete" ref="LIT1948Mashaf"></title>
+                          <textLang mainLang="gez"></textLang>
+                          <note>With many marginal notes, some of them in Amharic.</note>
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient757.xml:44]: Maybe you could refer to the addition and state that the catalogue is not clear on whether these are later additions

File: LondonBritishLibrary/orient/BLorient757.xml [hunk 4]
```diff
@@ -56,59 +159,90 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+                                <desc type="Unclear">Index of chapters to <ref target="#ms_i1">Maṣḥafa Mār Yǝṣhāq</ref>.</desc>
+                                <q xml:lang="gez"></q>
+                              </item>
+                              <item xml:id="a2">
+                                <locus from="3r"></locus>
+                                <desc type="GuestText">Sayings of Greek Philosophers on the death of
+                                <persName ref="PRS1666Alexande">Alexander the Great</persName>. Those
+                              mentioned by name are <persName ref="PRS2032Aristotl">Aristotle</persName>,
+                              Antiochus, Philemon
+                              <foreign xml:lang="gez">ፊልሞን፡</foreign>,
```
Comment (thea-m) [inline LondonBritishLibrary/orient/BLorient757.xml:205]: Maybe the Geez names could be in parantheses for better reading

File: LondonBritishLibrary/orient/BLorient757.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply LondonBritishLibrary/orient/BLorient757.xml]: Probably a difference between the number of actual written lines and the number of "line areas" (space for one written line delimited by two ruled lines) should be considered, if possible. If the scribe squeezed more written lines into the same number of line areas (as the result, the size of the script had to be diminished), this should be only commented, in my opinion.

File: LondonBritishLibrary/orient/BLorient757.xml [hunk 5]
```diff
@@ -56,59 +159,90 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
+
                         </objectDesc>
-                        
+
                         <handDesc>
-                            
+
                             <handNote script="Ethiopic" xml:id="h1">
-                                <desc>The manuscript is fairly written of the <date>18th century.</date>.
+                                <desc>The manuscript is fairly written.
```
Comment (DenisNosnitsin1970) [inline LondonBritishLibrary/orient/BLorient757.xml:179]: "fairly written" is a very old style expression. "Finely written"?

File: LondonBritishLibrary/orient/BLorient757.xml [hunk 6]
```diff
@@ -56,59 +159,90 @@ schematypens="http://relaxng.org/ns/structure/1.0"
...
-                            
+                            <list>
+                              <item xml:id="a1">
+                                <locus from="2v"></locus>
+                                <desc type="Unclear">Index of chapters to <ref target="#ms_i1">Maṣḥafa Mār Yǝṣhāq</ref>.</desc>
+                                <q xml:lang="gez"></q>
+                              </item>
+                              <item xml:id="a2">
+                                <locus from="3r"></locus>
+                                <desc type="GuestText">Sayings of Greek Philosophers on the death of
```
Comment (DenisNosnitsin1970) [inline LondonBritishLibrary/orient/BLorient757.xml:196]: philosophers?

File: LondonBritishLibrary/orient/BLorient757.xml (hunk 5 above)
Comment (eu-genia) [reply LondonBritishLibrary/orient/BLorient757.xml:179]: I would not correct Wright's English unless truly necessary...

**Pull request merged**